src/android/PurchasesPlugin.java

```diff
@@ -122,7 +122,7 @@ private void purchaseProduct(final String productIdentifier,
             oldSKU,
             prorationMode,
             googleIsPersonalizedPrice,
-            presentedOfferingIdentifier,
+            convertToPresentedOfferingContextMap(presentedOfferingIdentifier),
```

Comment:
Note that I'm not changing the TS interfaces (this package does not use `purchases-typescript-internal`). So this will be using the deprecated property `offeringIdentifier` which is part of what PHC returns.